packages/vue/src/components/organisms/SfTabs/SfTabs.stories.js

```diff
@@ -63,7 +63,6 @@ storiesOf("Organisms|Tabs", module)
         <pre>
           <code>
             import { SfTabs } from "@storefront-ui/vue"
-            import { SfTabs } from "@storefront-ui/vue"
```

Comment:
The whole info/summary section isn't used anymore. There are several PRs open (mainly  #491) which deletes this section. Please don't make changes to it anymore (merge conflicts).

Reply:
I've reverted that change